examples/plotting/file/color_sliders.py

```diff
@@ -0,0 +1,131 @@
+callback = Callback(args=dict(source=source), code="""
+	function componentToHex(c) {
+		var hex = c.toString(16);
+		return hex.length == 1 ? "0" + hex : hex;
+	}
+	function rgbToHex(r, g, b) {
+		return "#" + componentToHex(r) + componentToHex(g) + componentToHex(b);
+	}
+	var data = source.get('data');
+	var RS = red_slider;
+	var GS = green_slider;
+	var BS = blue_slider;
+	color = data['color'];
+	var red_from_slider = RS.get('value');
+	var green_from_slider = GS.get('value');
+	var blue_from_slider = BS.get('value');
+	text_color = data['text_color'];
+	color[0] = rgbToHex(red_from_slider, green_from_slider, blue_from_slider);
+	if ((red_from_slider > 127) || (green_from_slider > 127) || (blue_from_slider > 127)) {
+		text_color[0] = '#000000';
+	}
+	else {
+		text_color[0] = '#ffffff';
+	}
+	source.trigger('change');
+""")
+
+# create slider tool objects to control the RGB levels for first plot. Set callback function to allow refresh
+red_slider = Slider(start=0, end=255, value=255, step=1, title="R", callback=callback)
+green_slider = Slider(start=0, end=255, value=255, step=1, title="G", callback=callback)
+blue_slider = Slider(start=0, end=255, value=255, step=1, title="B", callback=callback)
+callback.args['red_slider'] = red_slider
+callback.args['green_slider'] = green_slider
+callback.args['blue_slider'] = blue_slider
+
+# plot 2: create a color spectrum with a hover-over tool to inspect hex codes
+brightness = 0.8 # at the moment this is hard-coded. Change if you want brighter/darker colors
+crx = range(1,1001) # the resolution is 1000 colors
+cry = [ 5 for i in range(len(crx)) ]
+crcolor, crRGBs = generate_color_range(1000,brightness) # produce spectrum
+
+# make data source object to allow information to be displayed by hover tool
+crsource = bkplt.ColumnDataSource(data=dict(x = crx, y = cry, crcolor = crcolor, RGBs = crRGBs))
+
+tools2 = 'reset, save, hover'
+
+# create second plot
+p2 = bkplt.figure(x_range=(0,1000), y_range=(0,10), plot_width=600, plot_height=150, tools=tools2, title = 'hover over color')
+color_range1 = p2.rect(x='x', y='y', width=1, height=10, color='crcolor', source=crsource)
+# set up hover tool to show color hex code and sample swatch
+hover = p2.select(dict(type=HoverTool))
+hover.tooltips = [
+				   ('color', '$color[hex, rgb, swatch]:crcolor'),
+				   ('RGB levels', '@RGBs')
+				 ]
+
+# get rid of axis details for cleaner look
+p1.ygrid.grid_line_color = None
+p1.xgrid.grid_line_color = None
+p1.axis.axis_line_color  = None
+p1.axis.major_label_text_color = None
+p1.axis.major_tick_line_color = None
+p1.axis.minor_tick_line_color = None
+
+p2.ygrid.grid_line_color = None
+p2.xgrid.grid_line_color = None
+p2.axis.axis_line_color  = None
+p2.axis.major_label_text_color = None
+p2.axis.major_tick_line_color = None
+p2.axis.minor_tick_line_color = None
+
+layout = bkplt.hplot(
+	vform(red_slider, green_slider, blue_slider),
+	vform(p1, p2)
+)
+
+bkplt.output_file("colorSliders.html")
```

Comment:
the name has to match the filename + .html, so our testing engine pick the correct one...
So, this has to be `"color_sliders.html"`